### ISSUE TITLE
Could not load ROM

### ISSUE THREAD

**Issue by TGManquito** [rafaelvcaetano/melonDS-android#130]
* **Device model:** LG-M700
* **Android version:** Android API 25
* **melonDS version:** Beta 1.2.2

**Problem:**  
Insert a small description of the problem.

**Stack trace:**  
```
me.magnum.melonds.ui.emulator.EmulatorActivity$f: Failed to load ROM: NDS_FAILED
	at me.magnum.melonds.ui.emulator.j$e$a$a.a()
	at f.a.b0.e.e.a.o()
	at f.a.s.a()
	at f.a.b0.e.b.e$a.d()
	at f.a.b0.e.b.l$a.d()
	at f.a.b0.e.b.j$a.run()
	at f.a.r$a.run()
	at f.a.b0.g.j.run()
	at f.a.b0.g.j.call()
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
	at java.lang.Thread.run(Thread.java:761)

```

**Configuration:**  
```
RomCrashContext(emulatorConfiguration=EmulatorConfiguration(useCustomBios=false, dsConfigDirectory=null/, dsiConfigDirectory=null/, internalDirectory=/data/user/0/me.magnum.melonds/files, fastForwardSpeedMultiplier=-1.0, useJit=false, consoleType=DS, soundEnabled=true, micSource=BLOW, firmwareConfiguration=FirmwareConfiguration(nickname=Player, message=Hello!, language=1, favouriteColour=0, birthdayMonth=1, birthdayDay=1), rendererConfiguration=RendererConfiguration(videoFiltering=LINEAR, threadedRendering=true)), romPath=/storage/emulated/0/Download/Pokemon - Edicion Plata SoulSilver (Spain).zip4833 - Pokemon - Edicion Plata SoulSilver (Spain).nds, sramPath=/storage/emulated/0/Download/Pokemon - Edicion Plata SoulSilver (Spain).zip4833 - Pokemon - Edicion Plata SoulSilver (Spain).sav)
```

**Comment by shabbirmahmud** [comment]
Try to put your ROM files in a different directory outside the Download folder. Don't forget to update the directory in the emulator settings.If it doesn't solve the problem then can you use the latest beta [v1.3.0](https://github.com/rafaelvcaetano/melonDS-android/releases/tag/beta-1.3.0) and open a new issue? The latest version does not fix the problem but includes additional info that will help to find the cause.

**Comment by rafaelvcaetano** [comment]
If your problem is not yet fixed, try the latest [Beta 1.4.0](https://github.com/rafaelvcaetano/melonDS-android/releases/latest) release. If the problem is not fixed, please open a new issue.
